core/flyout_horizontal.ts

```diff
@@ -267,29 +267,35 @@ export class HorizontalFlyout extends Flyout {
     for (let i = 0, item; (item = contents[i]); i++) {
       if (item.type === 'block') {
         const block = item.block;
-        const allBlocks = block!.getDescendants(false);
+
+        if (block === undefined || block === null) { {
```

Comment:
This PR isn't passing tests right now, and I suspect this double opening curly brace after the "if" may be the culprit!